typings/assets/index.d.ts

```diff
@@ -19,8 +19,11 @@ export type ImageName =
 
 export type ImagesList = Record<ImageName, ImageRequireSource> & { readonly [key: string]: ImageRequireSource };
 
-export namespace Assets {
-  const icons: Readonly<IconsList>;
-  const emojis: Readonly<EmojisList>;
-  const images: Readonly<ImagesList>;
-}
+export const Assets: {
+  icons: {
+    [k: string]: Readonly<IconsList>
+  };
```

Comment:
Can you explain this change? 
Is it because we have paths for our icons? 
as Assets.icons.**general**.iconName
Cause technically, we're allowing both. 
either 
`Assets.icons.iconName`
or nesting path
`Assets.icons.general.iconName`

Reply:
Exactly. Changed so that it should support both ways now